timesketch/frontend-ng/src/views/Graph.vue

```diff
@@ -0,0 +1,658 @@
+<!--
+Copyright 2023 Google Inc. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<template>
+  <v-container fluid>
+    <v-card flat class="pa-3 pt-0 mt-n3" color="transparent">
+      <v-card class="d-flex align-start mb-1" outlined>
+        <v-text-field
```

Comment:
Note: When searching for nodes in the graph only lower case seems to work. Even the nodes are sometimes shown in full caps. E.g. when I have a node `SYSTEM` and search for `SYSTEM` in the search field, it will not find the node. But when searching for `system` it finds `SYSTEM` as expected.